### PR TITLE
[DDW-658] UI - Delegation setup wizard steps 0 & 1 - Fixes

### PR DESCRIPTION
**Pull request by tomothespian**
This PR fix translations failures and choose wallet dropdown styles improvements.

## Screenshots

__Choose wallet step - cardano theme select options arrow color fix__
Choose wallet step - cardano theme select arrow

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test "translations" on Storybook stories `Staking->Stake Pools`
- [ ] Test "translations" on Storybook story `Staking->Epochs`
- [ ] Test "translations" on Storybook story `Staking->DelegationStepsIntroDialog`
- [ ] Test "translations" on Storybook story `Staking->DelegationStepsNotAvailableDialog`
- [ ] Test Storybook story `Staking->DelegationStepsChooseWalletDialog`

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@tomothespian please check why the BK build is failing 🙏

File: source/renderer/app/containers/staking/dialogs/DelegationSetupWizardDialogContainer.js [hunk 1]
```diff
@@ -37,22 +37,27 @@ const messages = defineMessages({
   },
 });
 
-type Props = InjectedContainerProps;
```
Comment (nikolaglumac) [inline source/renderer/app/containers/staking/dialogs/DelegationSetupWizardDialogContainer.js]: @tomothespian please revert this change 🙏

File: source/renderer/app/containers/staking/DelegationCenterPage.js [hunk 2]
```diff
@@ -2,16 +2,24 @@
 import React, { Component } from 'react';
 import { observer, inject } from 'mobx-react';
 import DelegationCenter from '../../components/staking/delegation-center/DelegationCenter';
+import DelegationSetupWizardDialogContainer from './dialogs/DelegationSetupWizardDialogContainer';
+import DelegationSetupWizardDialog from '../../components/staking/delegation-setup-wizard/DelegationSetupWizardDialog';
 import type { InjectedProps } from '../../types/injectedPropsType';
 
-type Props = InjectedProps;
```
Comment (nikolaglumac) [inline source/renderer/app/containers/staking/DelegationCenterPage.js]: @tomothespian please revert this change 🙏

**Comment by IuliaDolishniak** [comment]
Shouldn't these steps also be translated?
screen

**Comment by tomothespian** [comment]
> Shouldn't these steps also be translated?
> screen

@IuliaDolishniak Steps are translated but in parent container which is common for all steps dialogs.  Here, in storybook, I'm passing that to each dialog directly from storybook reading from translation file depending on selected locale and doing manual translate in story. If you change locale try to refresh page and you will see that also that is translated. Once Danilo's PR with localStorage locales will be merged I will also improve storybook to automatically change translations.

**Comment by IuliaDolishniak** [comment]
Yep, it works. Thanks!
